src/lib/gundb/UserStorageClass.js

```diff
@@ -387,7 +387,8 @@ export class UserStorage {
    * Object with default value for profile fields
    */
   profileDefaults: {} = {
-    mobile: '',
+    mobile: null,
```

Comment:
let's keep '' as default:
1. perform conversion to null on write field
2. on read field check value - if it's null -  apply value from profileDefaults